src/styles/PersistentBanner.module.less

```diff
@@ -0,0 +1,34 @@
+@import 'globalSettings.module.less';
+
+.banner {
+    position: fixed;
+    line-height: 20px;
+    padding-top: 6px;
+    background: #fff;
+    height: 31px;
+    width: 100%;
+    z-index: 101;
+    text-align: center;
+    font-size: 1rem;
+}
+
+// Keep this the same as the banner height
+.banner_whitespace{
+    height:31px;
```

Comment:
better if you change `pixel` to `rem` or `%` so that it scales according to browser